packages/installer.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,222 @@
+$ErrorActionPreference = 'Continue'
+$global:VerbosePreference = "SilentlyContinue"
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+    # Gather packages to install
+    $installedPackages = (VM-Get-InstalledPackages).Name
+    $configPath = Join-Path ${Env:VM_COMMON_DIR} "packages.xml" -Resolve
+    $configXml = [xml](Get-Content $configPath)
+    $packagesToInstall = $configXml.config.packages.package.name | Where-Object { $installedPackages -notcontains $_ }
+
+    # List packages to install
+    Write-Host "[+] Packages to install:"
+    foreach ($package in $packagesToInstall) {
+        Write-Host "`t[+] $package"
+    }
+    Start-Sleep 1
+
+    # Install the packages
+    try {
+        foreach ($package in $packagesToInstall) {
+            Write-Host "[+] Installing: $package" -ForegroundColor Cyan
+            choco install "$package" -y
+            VM-Write-Log "INFO" "Package $package has been installed"
+        }
+    } catch {
+        VM-Write-Log-Exception $_
+    }
+    VM-Write-Log "INFO" "[+] All packages complete"
+
+    # Set Profile/Version specific configurations
+    VM-Write-Log "INFO" "[+] Beginning Windows OS VM profile configuration changes"
+    VM-Apply-Configurations $(Join-Path $Env:VM_COMMON_DIR "config.xml")
```

Comment:
```suggestion
    VM-Apply-Configurations $(Join-Path $Env:VM_COMMON_DIR "config.xml" -Resolve)
```

Reply:
It's good to fail early and an easy way is to append "-Resolve" to "Join-Path" which will instruct it to verify the path exists